README.md

```diff
@@ -47,15 +47,17 @@ Legend:
 
 ## Specification Summary
 
-A Functions Framework instantiates web server and invokes function code in response to an HTTP request (`http`) or CloudEvent request (`cloudevent`) depending on the function's signature type. A Functions Framework may optionally support functions with signature type `event` for legacy-style events.
+A Functions Framework instantiates web server and invokes function code in response to an **HTTP** (`http`) or **CloudEvent** (`cloudevent`) request depending on the function's signature type. A Functions Framework may also optionally support functions with signature type `event` for legacy-style events.
 
 The Functions Framework library must be configurable via environment variables and may be configurable via command-line flags:
 
-Command-line flag         | Environment variable      | Description
-------------------------- | ------------------------- | -----------
-`--port`                    | `PORT`                    | The port on which the Functions Framework listens for requests. Default: `8080`
-`--target`         | `FUNCTION_TARGET`         | The name of the exported function to be invoked in response to requests. Default: `function`
-`--signature-type` | `FUNCTION_SIGNATURE_TYPE` | The signature used when writing your function. Controls unmarshalling rules and determines which arguments are used to invoke your function. Default: `http`; accepted values: `http` or `cloudevent`
+Required | Command-line flag | Environment variable | Description
+--- | --- | --- | ---
+YES | `--port` | `PORT` | The port on which the Functions Framework listens for requests. Default: `8080`
+NO | `--target` | `FUNCTION_TARGET` | The name of the exported function to be invoked in response to requests. Default: `function`
```

Comment:
I think that's a design decision we should probably discuss.

If only one function is registered with the framework, should we run that function? Or should we require a function to specify that function target?

Reply:
Hmm, we can discuss it for the future. Though for now, we have to leave it as required because otherwise the existing frameworks will not be compliant. 

Not having the function target could require significant changes to the buildpacks and the frameworks, and could be expensive performance wise in some languages.